Tests/Marketplace/marketplace_services.py

```diff
@@ -485,14 +518,7 @@ def _parse_pack_metadata(self, parse_dependencies: bool = False):
         Returns:
             dict: parsed pack metadata.
         """
-        pack_metadata = {
-            Metadata.CREATED: self._create_date,
-            Metadata.UPDATED: self._update_date,
-            Metadata.DOWNLOADS: self._downloads_count,
-            Metadata.TAGS: list(self._tags or []),
-            Metadata.SEARCH_RANK: self._search_rank,
-            Metadata.INTEGRATIONS: self._related_integration_images,
-        }
+        pack_metadata = self.update_metadata
```

Comment:
Since this code runs only for modified packs, we don't need to update the metadata again because it should be updated already.
Please revert.

Reply:
Using the function to prevent duplication of code, in case of modified packs (is_metadata_updated = False) it will not update the whole metadata again.

Reply:
and it is called from "format_metadata" which is called also when the pack is_metadata_updated